crates/astria-cli/README.md

```diff
@@ -0,0 +1,85 @@
+# Astria CLI
+
+Astria CLI is a command line tool for interacting with the Sequencer network
+and for managing your own rollup deployments. You can create a
+Sequencer account, check balances, generate rollup deployment configurations,
+deploy rollups, and more.
+
+## Dependencies
+
+* rust - <https://www.rust-lang.org/tools/install>
+* docker - <https://docs.docker.com/get-docker/>
+* kubectl - <https://kubernetes.io/docs/tasks/tools/>
+* kind - <https://kind.sigs.k8s.io/docs/user/quick-start/#installation>
+* helm - <https://helm.sh/docs/intro/install/>
+
+## Setup
+
+## Building
+
+```sh
+cargo build --release
+```
+
+## Running
+
+```sh
+# from monorepo root
+./target/release/astria-cli --help
+
+# examples:
+
+# create account on Sequencer
+./target/release/astria-cli sequencer account create
+
+# create a rollup config
+./target/release/astria-cli rollup config create \
+  --use-tty \
+  --log-level DEBUG \
+  --rollup.name somerollupname \
+  --rollup.chain-id somechainid \
+  --rollup.network-id 42 \
+  --rollup.skip-empty-blocks \
+  --sequencer.initial-block-height 1 \
```

Comment:
I think this command is partially documenting all config options, but generally probably don't want people to default to starting at first block